wire-ios/Wire-iOS/Sources/UserInterface/Location/LocationSelectionViewController.swift

```diff
@@ -105,6 +107,9 @@ final class LocationSelectionViewController: UIViewController {
             $0.translatesAutoresizingMaskIntoConstraints = false
         }
 
+        sendControllerHeightConstraint = sendController.heightAnchor.constraint(equalToConstant: 56 + UIScreen.safeArea.bottom)
```

Comment:
We should probably use the `safeAreaLayoutGuide` instead of `UIScreen` here, also below.

Reply:
Fixed it thanks!